src/utils/fees.ts

```diff
@@ -29,17 +13,23 @@ const FEE_OPTIONS = {
   CUSTOM: { name: 'Custom', label: 'Custom' },
 };
 
+const feePriceInWei = (asset: Asset, feePrice: number) => {
```

Comment:
The naming of this function is weird because that's not what it's doing
It's not producing a "wei" value, it's producing a normalized or unit value
so maybe call this `feePriceInUnit`